central/processindicator/datastore/datastore.go

```diff
@@ -36,6 +37,9 @@ type DataStore interface {
 
 	WalkAll(ctx context.Context, fn func(pi *storage.ProcessIndicator) error) error
 
+	// GetProcessIndicatorsRiskView retrieves minimal fields from process indicator for risk evaluation
+	GetProcessIndicatorsRiskView(ctx context.Context, q *v1.Query) ([]*views.ProcessIndicatorRiskView, error)
```

Comment:
It's used only in once place. How about instead of query accepting DeploymentId?